packages/react/src/createReactQueryHooks.tsx

```diff
@@ -343,8 +344,11 @@ export function createReactQueryHooks<
     ],
     opts: {
       enabled?: boolean;
-      error?: (err: TError) => void;
-      next: (data: TOutput) => void;
```

Comment:
Don't touch these, please

Reply:
So you're against the API change?

Should I just add start end stop callback so?

Reply:
Add stuff, don't modify existing.

Reply:
I think we are ok to remove these properties if following the design I propose here: https://github.com/trpc/trpc/pull/1999#issuecomment-1159030370. (👈 cc @sachinraja any thoughts?)

Regardless, it's probably best to let the new API evolve and settle before making these changes though.